specification/compute/resource-manager/Microsoft.Compute/stable/2021-03-01/compute.json

```diff
@@ -6888,6 +7342,305 @@
+        },
+        "sourceMetadata": {
+          "readOnly": true,
+          "$ref": "#/definitions/RestorePointSourceMetadata",
+          "description": "Gets the details of the VM captured at the time of creation of restore point."
+        },
+        "provisioningState": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Gets the provisioning state of restore point, which only appears in the response."
+        },
+        "consistencyMode": {
+          "type": "string",
+          "readOnly": true,
+          "enum": [
+            "CrashConsistent",
+            "FileSystemConsistent",
+            "ApplicationConsistent"
+          ],
+          "x-ms-enum": {
+            "name": "ConsistencyModeTypes",
+            "modelAsString": false
+          },
+          "description": "Gets or sets the consistency mode for the restore point."
+        },
+        "provisioningDetails": {
+          "$ref": "#/definitions/RestorePointProvisioningDetails",
+          "readOnly": true,
+          "description": "Gets the provisioning details set by the server during Create restore point operation."
+        },
+        "excludeDisks": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ApiEntityReference"
+          },
+          "readOnly": false,
+          "description": "List of disk resource ids that the customer wishes to exclude from the restore point. If no disks are specified, all disks will be included."
+        }
+      },
+      "description": "The restore point properties."
+    },
+    "RestorePointSourceMetadata": {
+      "properties": {
+        "hardwareProfile": {
+          "$ref": "#/definitions/HardwareProfile",
+          "description": "Gets the hardware profile."
+        },
+        "storageProfile": {
+          "$ref": "#/definitions/RestorePointSourceVMStorageProfile",
+          "description": "Gets the storage profile."
+        },
+        "osProfile": {
+          "$ref": "#/definitions/OSProfile",
+          "description": "Gets the OS profile."
+        },
+        "diagnosticsProfile": {
+          "$ref": "#/definitions/DiagnosticsProfile",
+          "description": "Gets the diagnostics profile."
+        },
+        "licenseType": {
+          "type": "string",
+          "description": "Gets the license type, which is for bring your own license scenario."
+        },
+        "vmId": {
+          "type": "string",
+          "description": "Gets the virtual machine unique id."
+        }
+      },
+      "description": "Describes the properties of a Virtual Machine from which the restore point was created."
+    },
+    "RestorePointSourceVMStorageProfile": {
+      "properties": {
+        "osDisk": {
+          "$ref": "#/definitions/RestorePointSourceVMOSDisk",
+          "description": "Gets the OS disk of the VM captured at the time of creation of restore point."
```

Comment:
Gets the OS disk of the VM captured at the time of the restore point creation.

Reply:
Same comment applies to description of RestorePointSourceVMDataDisk

Reply:
done